client/src/about/about.yaml

```diff
@@ -121,59 +119,77 @@ our_data_sub_title_2:
 
 behind_scenes_title:
   en: Behind the Scenes
-  fr: Derrière la scène
+  fr: Derrière les coulisses
 
 behind_scenes_sub_title_1:
   en: What technologies are used in GC InfoBase?
-  fr: Quelles sont les technologies utilisées dans InfoBase du GC?
+  fr: Quelles sont les technologies utilisées dans l'InfoBase du GC?
 
 behind_scenes_sub_text_1:
   transform: [handlebars]
   en: |
     GC InfoBase is a single-page web application, built in-house and from scratch. It was developed using open-source libraries and standards (such as React, Bootstrap, webpack, Sass, and GraphQL).
-    The team keeps the iteration time between prototypes and production short. Having built it from the ground-up gives us the flexibility to experiment 
-    with new ways of displaying information and allows GC InfoBase to be constantly evolving, meeting user’s growing needs and expectations.
+    We keep the iteration time between prototypes and production short. Having built it from the ground-up gives us the flexibility to experiment 
+    with new ways of displaying information and allows GC InfoBase to constantly evolve, meeting users' growing needs and expectations.
   fr: |
-    InfoBase du GC est une application Web-maison d’une seule page montée de toutes pièces. Elle a été développée à l’aide de normes et de bibliothèques logicielles en libre accès 
-    (comme React, Bootstrap, webpack, Sass, et GraphQL). L’équipe maintient un court délai d’itération entre les prototypes et la production. Comme la page est montée de toutes 
-    pièces, nous sommes en mesure de mettre à l’essai de nouvelles présentations de l’information et de constamment faire évoluer InfoBase du GC pour répondre aux attentes et besoins 
+    L'InfoBase du GC est une application Web-maison d’une seule page montée de toutes pièces. Elle a été développée à l’aide de normes et de bibliothèques logicielles en libre accès 
+    (comme React, Bootstrap, webpack, Sass, et GraphQL). Nous avons un bref délai d’itération entre les prototypes et la production. Comme la page est montée de toutes 
+    pièces, nous sommes en mesure de mettre à l’essai de nouvelles présentations de l’information et de constamment faire évoluer l'InfoBase du GC pour répondre aux attentes et besoins 
     croissants des utilisateurs.
 
 behind_scenes_sub_title_2:
+  en: HOW DOES THE INFORMATION ON GC INFOBASE DIFFER FROM OFFICAL REPORTS?
+  fr: COMMENT LES RENSEIGNEMENTS DANS L'INFOBASE DU GC DIFFÈRENT-ILS DES RAPPORTS OFFICIELS?
+
+behind_scenes_sub_text_2:
+  transform: [handlebars, markdown]
+  en: |
+    Whenever possible, we align the information on GC InfoBase to the universe of organizations and concepts that are presented in Volume II of the {{gl_tt "Public Accounts" "PA"}}. As a result, organizations that are presented separately in other reports may be grouped together on GC InfoBase to match the {{gl_tt "Public Accounts" "PA"}}.
```

Comment:
I think gl_tt once per section of the same word would be pretty reasonable so that there is just one PA gl_tt for behind_scenes, once for our_data